python-avd/pyavd/_eos_designs/structured_config/network_services/router_bgp.py

```diff
@@ -324,37 +325,67 @@ def _update_router_bgp_vrf_mlag_neighbor_cfg(
         interface_name = f"Vlan{vlan_id}"
 
         if self.inputs.underlay_rfc5549 and self.inputs.overlay_mlag_rfc5549:
-            bgp_vrf.neighbor_interfaces.append_new(
-                name=interface_name,
-                peer_group=self.shared_utils.mlag_vrfs_peer_group_name,
-                remote_as=self.shared_utils.formatted_bgp_as,
-                description=AvdStringFormatter().format(
-                    self.inputs.mlag_bgp_peer_description,
-                    mlag_peer=self.shared_utils.mlag_peer,
-                    interface=interface_name,
-                    peer_interface=interface_name,
+            if isinstance(bgp_vrf, EosCliConfigGen.RouterBgp.VrfsItem):
```

Comment:
I guess you are doing this for type checking - lets do this in another PR (and think if can find a better way as this looks like a lot of code suddenly)